### ISSUE TITLE
Flaky spec: Voter Origin Trying to vote in booth and then in web

### ISSUE THREAD

**Issue by bertocq** [AyuntamientoMadrid/consul#1195]
Warning!
===
This spec probably only fails between 00:00 and 02:00 am Madrid time

# What
Tests that fail randomly are called "flakies", this one seems to be one:

**Randomized seed:** 13457

**Failure:**

```
 1) Voter Origin Trying to vote the same poll in booth and web Trying to vote in booth and then in web
     Failure/Error: select 'DNI', from: 'residence_document_type'
     Capybara::ElementNotFound:
       Unable to find visible select box "residence_document_type" that is not disabled
     # ./spec/support/common_actions.rb:204:in `officing_verify_residence'
     # ./spec/support/common_actions.rb:496:in `vote_for_poll_via_booth'
     # ./spec/features/polls/voter_spec.rb:142:in `block (4 levels) in <top (required)>'
```

# How
- [ ] Explain why the test is flaky, or under which conditions/scenario it fails randomly
- [ ] Explain why your PR fixes it
- [ ] Create a backport PR to consul/consul when the fixing PR is approved

### Random values issues
If the problem comes from randomly generated values, running multiple times a single spec could help you reproduce the failure by running at your command line:
```bash
for run in {1..10}
do
  bin/rspec ./spec/features/budgets/investments_spec.rb:256
done
```

You can also try running a single spec in Travis:
Add option `:focus` to the spec and push your branch to Github, for example:
```ruby
scenario 'Show', :focus do
```

But remember to remove that `:focus` changes afterwards when submitting your PR changes!

### Test order issues
Running specs in the order they failed may discover that the problem is that a previous test sets an state in the test environment that makes our flaky fail/pass. Tests should be independent from the rest.

After executing rspec you can see the seed used, add it as an option to rspec, for example:
`bin/rspec --seed 55638` (check **Randomized seed** value at beginning of issue)

### Other things to watch for
- Time related issues (current time, two time or date comparisons with miliseconds/time when its not needed)
- https://semaphoreci.com/community/tutorials/how-to-deal-with-and-eliminate-flaky-tests

**Comment by iagirre** [comment]
I'll go with this.

**Comment by raul-fuentes** [comment]
I'll backport this one

**Comment by raul-fuentes** [comment]
This change it's  already on master, backport not needed

**Comment by javierv** [comment]
@raul-fuentes I found this issue while working on consul#2712. I haven't seen any reference to the stubs added by #1342 while checking [the voter spec file in consul](https://github.com/consul/consul/blob/master/spec/features/polls/voter_spec.rb), and the affected spec still fails on my machine at midnight. Is it possible that it seems somehow to be backported but actually it's not :thinking:?

P.S. Thank you for backporting #1349 :+1:!

**Comment by javierm** [comment]
Closed via consu#2712 and #1625.
